lms/djangoapps/teams/static/teams/js/spec/views/teams_tab_spec.js

```diff
@@ -38,14 +38,30 @@ define([
                 ));
         };
 
-        var createTeamsTabView = function(test, options) {
-            var teamsTabView = new TeamsTabView(
+        const verifyTeamsetTeamsRequest = () => {
+            AjaxHelpers.expectRequestURL(
+                requests,
+                TeamSpecHelpers.testContext.teamMembershipsUrl,
+                {
+                    username: TeamSpecHelpers.testUser,
+                    course_id: TeamSpecHelpers.testCourseID,
+                    teamset_id: TeamSpecHelpers.testTopicID,
+                }
+            );
+            AjaxHelpers.respondWithJson(
+                requests,
+                TeamSpecHelpers.createMockTeamsResponse({results: []})
+            );
+        };
+
+        const createTeamsTabView = (options) => {
+            const teamsTabView = new TeamsTabView(
                 {
                     el: $('.teams-content'),
                     context: TeamSpecHelpers.createMockContext(options)
                 }
             );
-            requests = AjaxHelpers.requests(test);
```

Comment:
All of these changes are because the referenced method doesn't actually expect or do anything with this argument